crates/pipeline_manager/src/api.rs

```diff
@@ -101,6 +104,13 @@ We also support directly ingesting and consuming data via HTTP;
 see the `pipelines/{pipeline_id}/ingress` and `pipelines/{pipeline_id}/egress`
 endpoints.
 
+* *External service*. An external service with a unique name and ID.
```

Comment:
This documentation is duplicated in the SQL schema. It's better to only keep this version here and simplify the description in the schema.

Reply:
Simplified description in schema.